tests/main/snap-user-service/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: Check that snap user services are installed and started.
+
+systems:
+    # Ubuntu 14.04's systemd doesn't have user@.service
+    - -ubuntu-14.04-*
+    # Amazon Linux 2 gives error "Unit user@12345.service not loaded."
+    - -amazon-linux-2-*
+    # Centos 7 gives error "Unit user@12345.service not loaded."
+    - -centos-7-*
+
+kill-timeout: 5m
+
+prepare: |
+    snap set system experimental.user-daemons=true
+    session-tool --kill-leaked
```

Comment:
You need `session-tool --prepare` as well.

Reply:
In this test, I'm installing a snap while no session is running, then verifying that it is started when a new user session is started.  So the `--prepare` call occurs within the `execute:` block.

Reply:
Ah, I missed this! Thank you.